lib/tbot/config/configtemplate_ssh_host_cert.go

```diff
@@ -31,6 +33,16 @@ const (
 	// defaultSSHHostCertPrefix is the default filename prefix for the SSH host
 	// certificate
 	defaultSSHHostCertPrefix = "ssh_host"
+
+	// sshHostCertSuffix is the suffix appended to the generated host certificate.
+	sshHostCertSuffix = "-cert.pub"
+
+	// sshHostCASuffix is the suffix appended to the user CA file.
+	sshHostCASuffix = "-user-ca.pub"
```

Comment:
Surely this should be `sshUserCASuffix` ?

Reply:
It is admittedly confusing but the var name prefix is more meant to notate that the variable is for the `sshHost` config template, so `(sshHost)(CASuffix)`. I'll rename it to clarify.